django/contrib/auth/views.py

```diff
@@ -15,9 +13,12 @@
     SetPasswordForm,
 )
 from django.contrib.auth.tokens import default_token_generator
+
+# For backwards compatibility with Django < 5.1
+from django.contrib.auth.utils import redirect_to_login  # NOQA: F401
```

Comment:
I'm wondering if we have to do this, if so we should have a deprecation message.
Looking at this PR which is a little similar (https://github.com/django/django/pull/17050) looks like we don't necessarily need to. 🤔